src/test/resources/com/znsio/sample/e2e/features/amazon_cart.feature

```diff
@@ -0,0 +1,10 @@
+@amazon
+Feature: Amazon
+
+  @web @amazon @amazon-test
+  Scenario: Add to cart
+    Given I launch amazon application
+    And I search for "Iphone 13" and verify results
+    When I Open first search result details
+    And I add above item to cart
+    Then I verify details in cart
```

Comment:
Verification step is not required. It should be done automatically as part of the earlier step

Reply:
Acknowledge